ui/redux/actions/claims.js

```diff
@@ -659,7 +664,41 @@ export function doClaimSearch(
       return false;
     };
 
-    return await Lbry.claim_search(options).then(success, failure);
+    const autoPaginate = () => {
+      let allClaims = [];
+
+      const next = async (data: ClaimSearchResponse) => {
+        allClaims = allClaims.concat(data.items);
+
+        const moreData = data.items.length === options.page_size;
+
+        options.page++;
+
+        if (options.page > 3 || !moreData) {
+          // Flow doesn't understand that page_size is an optional property with a guaranteed default value.
+          // $FlowFixMe
+          return success({
+            items: allClaims,
+            page: options.page,
+            page_size: options.page_size,
+            total_items: allClaims.length,
+            total_pages: options.page,
+          });
```

Comment:
@infinite-persistence @tzarebczan Do you know why `ClaimSearchResponse` declares `total_items` and `total_pages` as part of the response type for claim search?

I don't see those ever being returned by the SDK. Also I'm not sure how `total_pages` really makes any sense in this context?

Reply:
It's probably some older code when we used to, but we had to remove those via the --no_totals param for performance reasons. Should be safe to remove.